Testnet v3 docs (#109)
Manta-Network/docs: docs/guides/DolphinPay.md

# 🐬 Try MantaPay on Dolphin Testnet

[The Dolphin zkTransact page](https://app.manta.network/dolphin/transact) allows users to privatize public assets to private, transfer private asset, and convert private asset back to public assets (as demonstrated below).

<div style={{textAlign: 'center'}}>
    <img alt="DolphinPay Overview" src="/img/guides/DolphinPay.svg" width="70%"/>
</div>

Note: Public assets and private assets have different addresses (public addresses vs zkAddresses). There is no relationship between a public address and a zkAddress.

## Setup Testnet Accounts

### Install a Browser Wallet and Create a Public Address

Currently, Dolphin Testnet V3 supports Talisman, SubWallet, Polakdot.js wallets to manage public assets.

To get started, click the "Connect Wallet" button.

<br/>

   <div style={{textAlign: 'center'}}>
    <img alt="signer-init" src="/img/guides/connect_wallet_button.png" width="80%"/>
   </div>

<br/>

> If you don't already have a wallet installed, click the "Install" link for your your chosen wallet. Once you install the wallet, it will give you instructions to set up your first account. When you are finished, refresh the page.

Reviewer: after "refresh the page", I think it will be great to add a little more, "When you are finished, refresh the page and the button text will change from Install to Connect"

Reviewer: fixed

<br/>

   <div style={{textAlign: 'center'}}>
    <img alt="signer-init" src="/img/guides/signer_connected.png" width="80%"/>
   </div>

   <div style={{textAlign: 'center'}}>
    <img alt="signer-init" src="/img/guides/signer_not_connected.png" width="80%"/>
   </div>
<br/>

If you don't have signer installed, you can find the download link by clicking on the "Connect Signer" button.

<br/>

   <div style={{textAlign: 'center'}}>
    <img alt="signer-init" src="/img/guides/signer_install_modal.png" width="50%"/>
   </div>
<br/>

Open Manta Signer and follow instructions to create your zkAddress. See our [guide to Manta Signer](https://docs.manta.network/docs/guides/MantaSigner) for more details.

<br/>

   <div style={{textAlign: 'center'}}>
    <img alt="signer-init" src="/img/guides/signer_setup_page.png" width="50%"/>
   </div>
<br/>

### Get Testnet Tokens

Join [Manta & Calamari's Discord](https://t.co/5BacMMLSCW), navigate to the `#dolphin-faucet` channel, and type `/gimme`. You should see a faucet options prompt:

<br/>
   <div style={{textAlign: 'center'}}>
    <img alt="faucet" src="/img/guides/faucet_bot.png" width="70%"/>
   </div>
<br/>

> Enter your polkadot address and claim some `DOL`. Feel free to claim other testnet tokens like `KSM`, `KAR`, `MOVR`, but you will have to bridge them onto Dolphin (see our guide to bridging [here](https://docs.manta.network/docs/guides/DolphinXCMBridge)). If you want to do private transactions with bridged coins, you will still need `DOL` to pay fees. Note that you can finish this guide with only `DOL`.

Reviewer: https://docs.manta.network/docs/guides/DolphinXCMBridge
change to 
https://docs.manta.network/docs/guides/DolphinBridge

Reviewer: fixed

# 🐬 Try MantaPay on Dolphin Testnet

[The Dolphin zkTransact page](https://app.manta.network/dolphin/transact) allows users to privatize public assets to private or
"zk" assets, transfer zk assets, and convert zk assets back to public assets.

> The most important concept to remember when using the Dolphin zkTransact page is that public assets and zk assets have different addresses: public addresses and ZkAddresses. There is no relationship between a public address and a ZkAddress.

Reviewer 2: i think it'd look nice as `zkAddress/es` throughout the whole doc

To get started, click the "Connect Wallet" button.

<br/>

   <div style={{textAlign: 'center'}}>
    <img alt="connect_wallet_button" src="/img/guides/connect_wallet_button.png" width="80%"/>
   </div>

<br/>

If you don't already have a wallet installed, click the "Install" link for your chosen wallet. Once you install the wallet, it will give you instructions to set up your first account. When you are finished, refresh the page and the button text will change from "Install" to "Connect".

<br/>

   <div style={{textAlign: 'center'}}>
    <img alt="connect_wallet_modal" src="/img/guides/connect_wallet_modal.png" width="50%"/>
   </div>

<br/>

Once you have a wallet installed, click the "Connect Wallet" button, press "Connect", and follow the prompts from your wallet extension. When you have successfully connected, you will see the wallet in the navbar.

<br/>

   <div style={{textAlign: 'center'}}>
    <img alt="wallet_connected" src="/img/guides/wallet_connected.png" width="80%"/>
   </div>
<br/>

### Install [Manta Signer](https://signer.manta.network) and Create a zkAccount

> Manta Signer is a native app to turbo-charge zero-knowledge proof generation.It allows you to send and receive private payments. Signer is available for:

Reviewer 2: ```suggestion
Manta Signer is a native app to turbo-charge zero-knowledge proof generation. It allows you to send and receive private payments. Signer is available for:
```

   <div style={{textAlign: 'center'}}>
    <img alt="signer_connected" src="/img/guides/signer_connected.png" width="80%"/>
   </div>

   <div style={{textAlign: 'center'}}>
    <img alt="signer_not_connected" src="/img/guides/signer_not_connected.png" width="80%"/>
   </div>
<br/>

If you don't have signer installed, you can find the download link by clicking on the "Connect Signer" button.

<br/>

   <div style={{textAlign: 'center'}}>
    <img alt="signer_install_modal" src="/img/guides/signer_install_modal.png" width="50%"/>
   </div>
<br/>

Open Manta Signer and follow instructions to create your ZkAddress. See our [guide to Manta Signer](https://docs.manta.network/docs/guides/MantaSigner) for more details.

<br/>

   <div style={{textAlign: 'center'}}>
    <img alt="signer_setup_page" src="/img/guides/signer_setup_page.png" width="50%"/>
   </div>
<br/>

### Get Testnet Tokens

Join [Manta & Calamari's Discord](https://t.co/5BacMMLSCW), navigate to the `#dolphin-faucet` channel, and type `/gimme`. You should see a faucet options prompt:

<br/>
   <div style={{textAlign: 'center'}}>
    <img alt="faucet_bot" src="/img/guides/faucet_bot.png" width="70%"/>
   </div>
<br/>

> Enter your polkadot address and claim some `DOL`. Feel free to claim other testnet tokens like `KSM`, `KAR`, `MOVR`, but you will have to bridge them onto Dolphin (see our guide to bridging [here](https://docs.manta.network/docs/guides/DolphinBridge)). If you want to do private transactions with bridged coins, you will still need `DOL` to pay fees. Note that you can finish this guide with only `DOL`.

Reviewer 2: ```suggestion
Enter your polkadot address and claim some `DOL`. Feel free to claim other testnet tokens like `KSM`, `KAR`, `MOVR`, to test cross-chain bridging of assets onto Dolphin (see our guide to bridging [here](https://docs.manta.network/docs/guides/DolphinBridge)). If you want to do private transactions with bridged coins, you will still need `DOL` to pay fees. Note that you can finish this guide with only `DOL`.
```

   <div style={{textAlign: 'center'}}>
    <img alt="to_private" src="/img/guides/to_private.png" width="50%"/>
   </div>
   <br/>
   Your browser wallet will prompt you to approve the transfer, and then publish it to the Dolphin blockchain:

<br/>
<br/>
   <div style={{textAlign: 'center'}}>
    <img alt="private_transfer_publish" src="/img/guides/private_transfer_publish.png" width="30%"/>
   </div>
<br/>
<br/>

### Send Private Assets to a ZkAddress
   Press the private-public toggle button at the top of the form, so that you can send tokens from your ZkAddress instead of your public address.
   <div style={{textAlign: 'center'}}>
    <img alt="toggle_public_private" src="/img/guides/toggle_public_private.png" width="50%"/>
   </div>
   <br/>

   Select the token you want to send, enter the amount you are sending, and enter the destination ZkAddress. If you need an example ZkAddress to send to, you can send to yourself (click on the copy icon next to your ZkAddress), or to this example ZkAddress:

   `DfZHqhfhtvjSxGY3c6UBnfDAPYCyF8xz9G4bHTHfCrYE`

>    ⚠️ Note that ZkAddresses and public addresses have a different format, and are not interchageable! You cannot send a private transfer to a public address, or a public transfer to a private address.

Reviewer 2: ```suggestion
:::note
zkAddresses and public addresses have a different format, and are not interchageable! You cannot send a private transfer to a public address, or a public transfer to a private address.
:::
```

  If your transaction is taking a long time, make sure Manta Signer and your browser wallet aren't awaiting approvals; you might not have sent actually the transaction. Otherwise, if your transaction keeps you waiting for more than a few minutes, please [file a bug report.](https://discord.gg/c72QMWEVyY)

5. How does private payment work?

   [Find out more here.](../learn/PrivatePayment.md)

6. How do I find my Calamari address on polkadot.js?

   Click on polkadot.js extension, then click on three vertical dots next to your address, and then select "Calamari Parachain." If you don't see the "Calamari Parachain" option, update your extensions metadata [here](https://polkadot.js.org/apps/#/settings/metadata).

7. How can I delete my Manta Signer Account?

   See our [guide to Manta Signer](https://docs.manta.network/docs/guides/MantaSigner) for more details.

8. How long will the testnet run?

>     The testnet will run forever 👾

Reviewer 2: i don't think this is accurate. I suggest
" We will run various iterations of testnets over time and reset the network (after announcement) when necessary. Do not expect your testnet assets to be available forever.
